flybywiresim/a32nx
# fix(model): delete redundant views #12+#13, add back old views #14+#15

Fixes #6157

## Summary of Changes
PR #6107 made old views 12 and 13 redundant. It also consumed the engine/wing views 14 and 15.
This PR removes redundant views 12 and 13 and adds the 2 missing views, 14 and 15, in those slots.

## Screenshots (if necessary)
<img src="https://user-images.githubusercontent.com/433429/140771788-0db0c6ad-aa05-4f61-832e-e7de3f5cef5d.jpg" width="50%"><img src="https://user-images.githubusercontent.com/433429/140771949-ef719424-3ad1-4375-9091-d68ce8c42651.jpg" width="50%">
<p align="right"><i>View 12 and 13, external engine views<i/></p>
<img src="https://user-images.githubusercontent.com/433429/140772263-93169f9e-8047-4532-8648-1b2773c384d0.jpg" width="50%"><img src="https://user-images.githubusercontent.com/433429/140772305-ab708929-56b2-4cb0-8f4c-b12b4dda0603.jpg" width="50%">
<p align="right"><i>View 14 and 15, engines, in front of wing<i/></p>
<img src="https://user-images.githubusercontent.com/433429/140772548-b3339c8a-c0d1-4384-bf96-5e13d24ddb37.jpg" width="50%"><img src="https://user-images.githubusercontent.com/433429/140772560-8c18f788-d3a5-46fa-b749-6a61dceaab73.jpg" width="50%">
<p align="right"><i>View 16 and 17, wing, above the wing<i/></p>
<img src="https://user-images.githubusercontent.com/433429/140772774-73f005ef-3240-467d-a660-f90bf5300186.jpg" width="50%"><img src="https://user-images.githubusercontent.com/433429/140772796-82cc21ea-1d99-4076-b558-e5757a2c09e1.jpg" width="50%">
<p align="right"><i>View 18 and 19, wing, behind the wing<i/></p>

## Additional context



Discord username (if different from GitHub): @bouveng

## Testing instructions
1. Spawn
2. Verify views 12+13 as above
3. Verify remaining 6 window views


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
